Jenkinsfile

```diff
@@ -13,22 +13,32 @@ pipeline {
 
     environment {
       INMANTA_TEST_ENV="${env.WORKSPACE}/env"
+      INMANTA_SERVER_PATH="${env.WORKSPACE}/server"
+      INMANTA_PYTHON_PATH="${env.WORKSPACE}/env/bin/python3"
```

Comment:
similar here:
```suggestion
      INMANTA_LS_PYTHON_PATH="${env.WORKSPACE}/env/bin/python3"
```

Reply:
(similar to https://github.com/inmanta/vscode-inmanta/pull/33#discussion_r515816250)

Reply:
For that one I am less sure, I was referring to the setting: `inmanta.pythonPath`

Reply:
Which is the python path of the environment where the inmanta language server is (or should be) installed.

Reply:
Is it better like this? https://github.com/inmanta/vscode-inmanta/pull/33/commits/7a93af6a1916cadbef688f0eaf529b7450e48659